source/types.ts

```diff
@@ -164,6 +164,12 @@ export type Store = {
 	shutdown?: () => Promise<void> | void
 }
 
+export enum ValidationLevel {
+	Off,
+	Warn,
+	Throw,
+}
```

Comment:
I'm not sure if this belongs in this file, since it's not actually a type. But I need to reference it in the Options type and this prevented me from needing circular imports.

Reply:
The original purpose of this file was to store all public facing types, interfaces and enums, so I'd say it very much does belong here.

Maybe we could change the file name?

Reply:
🤷 it's probably fine. TS enums are kind of weird in that they're both a type and compile actual JS code. Nearly everything else in TS is one or toe other, rather than both.